config/routes.rb

```diff
@@ -1,3 +1,9 @@
 Rails.application.routes.draw do
-  # For details on the DSL available within this file, see https://guides.rubyonrails.org/routing.html
+  namespace :api do
+    namespace :v1 do
+      namespace :public do
+        resources :companies,                    only: [:index]
```

Comment:
```suggestion
        resources :companies, only: [:index]
```

what happened here :D